code/modules/mob/mob_grab.dm

```diff
@@ -166,14 +166,14 @@
 						qdel(src)
 						return
 					state = GRAB_KILL
-					assailant.visible_message("<span class='danger'>[assailant] has tightened \his grip on [affecting]'s neck!</span>")
+					assailant.visible_message("<span class='danger'>[assailant] has tightened \his grip on [affecting]'s neck!</span>", "<span class='danger'>You tighten your grip on [affecting]'s neck!</span>")
 					add_logs(assailant, affecting, "strangled")
 
 					assailant.changeNext_move(CLICK_CD_TKSTRANGLE)
 					affecting.losebreath += 1
 				else
 					if(assailant)
-						assailant.visible_message("<span class='warning'>[assailant] was unable to tighten \his grip on [affecting]'s neck!</span>")
+						assailant.visible_message("<span class='warning'>You're unable to tighten your grip on [affecting]'s neck!</span>")

```

Comment:
Shit. Sorry I don't know how to test these cases for how anyone else sees it. At least now I (think I) know the difference between user.visible_message and user <<...